packages/frontend/src/pages/bridges-tvl/index.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 
 import { PageWrapper } from '../../components'
 import { BridgesTvlPage } from './BridgesTvlPage'
-import { getProps } from './getProps'
+import { getProps } from './props'
 
 export function getBridgesTvlPage(bridges: Bridge[], apiMain: ApiMain) {
   const { props, wrapper } = getProps(bridges, apiMain)
```

Comment:
in `line 12` there is a typo, should be `tvl`, I do not know hot to add comment to it ;p